test/oauth-server/config.yaml

```diff
@@ -0,0 +1,14 @@
+log:
+  debug: false
+apis:
+  http:
+    address: "0.0.0.0:9100"
+    tls:
+      caPool: "/secrets/public/rootca.crt"
+      keyFile: "/secrets/private/cert.key"
+      certFile: "/secrets/public/cert.crt"
+      clientCertificateRequired: true
+oauthSigner:
+  idTokenKeyFile: "/secrets/private/idToken.key"
```

Comment:
Why are you signing tokens with different certs?

Reply:
becase idToken must by sigend by RSA (ui library doesn't supports ECDSA)
and accessToken we signed by ECDSA (is better and smaller)

Reply:
Why exactly is ECDSA better if the oauth-server is used only in tests and bundle (example for development)? Why we make deployment and configuration more complex?